app/src/main/java/com/example/fittracker/activity/UserSettingsActivity.java

```diff
@@ -26,38 +25,41 @@ protected void onCreate(Bundle savedInstanceState) {
         init();
     }
 
-    public void init(){
-        presenter = new UserSettingsPresenter(new UserSettingsView(
-                this),new UserSettingsModel(UserRoomDatabase.getDatabase(this)));
-    }
+    public void init() {
+        presenter = new UserSettingsPresenter(new UserSettingsView(this)
+                ,new UserSettingsModel(UserRoomDatabase.getDatabase(this))
+                ,getSharedPreferences(ConstantUtils.USER_PREFERENCES, MODE_PRIVATE)
+                .getInt(ConstantUtils.USER_PREFERENCES_ID,ConstantUtils.ZERO));
+        presenter.onCreate();
```

Comment:
change the name of this method, call it init()

Reply:
Done